doc/readme/EXAMPLES.md

```diff
@@ -38,7 +38,7 @@ For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py`
 It is extremely simple to add a new example: simple create another `*.sg` file.
 We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
 
-If you porting an example is great, even better is when it comes with integration testing data of the numerical output, as described in [DEVELOPING.md](DEVELOPING.md#testing).
+If you are porting a new example, great! Better yet would be to add the [data](https://github.com/shogun-toolbox/shogun-data) so that we can do integration tests automatically as described in [DEVELOPING.md](DEVELOPING.md#testing).
```

Comment:
This should be “Better yet, add the data...” imo

Reply:
That good too, any other corrections?